lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -218,7 +232,96 @@ private function relationshipExists($sourceActorId, $targetActorId, $relationTyp
       ':type_id' => $relationTypeId
     ];
 
-    return QubitPdo::fetchOne($sql, $params) !== false
-      || QubitPdo::fetchOne($sql, $paramsVariant) !== false;
+    if ($results = QubitPdo::fetchAll($sql, $params, ['fetchMode' => PDO::FETCH_ASSOC]))
+    {
+      return $results[0]['id'];
+    }
+
+    $results = QubitPdo::fetchAll($sql, $paramsVariant, ['fetchMode' => PDO::FETCH_ASSOC]);
+
+    return count($results) ? $results[0]['id'] : false;
```

Comment:
I think you could use [QubitPdo::fetchColumn()](https://github.com/artefactual/atom/blob/qa/2.6.x/lib/QubitPdo.class.php#L62) to return the id column value, or false if there are no results?

Reply:
Ah good call... I always forget that exists. Thanks!